config/config/sc-config.yaml

```
@@ -80,11 +80,7 @@ user:
           memory: 100Mi
   # Todo remove dependencie on alertmanager from service cluster
   alertmanager:
-    group_by:
-      - cluster
-      - alertname
-      - severity
-
+    group_by: ['...']
```

Any reason why not to use a yaml list formatting?
I feel like yq will just convert this to a yaml list eventually.
```suggestion
    group_by:
      - '...'
```